orttraining/orttraining/python/training/_utils.py

```diff
@@ -22,7 +38,7 @@ def dtype_torch_to_numpy(torch_dtype):
         # NOTE: numpy doesn't support bfloat16
         return np.float16
     elif torch_dtype == torch.int64 or torch_dtype == torch.long:
-        return np.int64
+        return np.longlong # np.int64 doesn't work!?
```

Comment:
NIP: should we delete this comment?

Reply:
I believe this comment will save from people to try to use np.int64 to map a torch.int64.